Examples/FibonacciExecutor.py

```diff
@@ -14,24 +16,41 @@ def calc_fib(message):
     return a
 
 
-def handle_message(message):
-    # return calc_fib(message)
-    print("Fibonacci calc request {}".format(message))
+@gen.coroutine
+def handle_message(logger, message):
+    logger.info("Fibonacci calc request {}".format(message))
     res = calc_fib(message)
-    print("Fibonacci calc result {}".format(res))
-    f = asyncio.Future()
-    f.set_result(res)
-    return f
+    logger.info("Fibonacci calc result {}".format(res))
+    return res
 
 
 if __name__ == "__main__":
     if sys.platform == 'win32':
         asyncio.set_event_loop_policy(asyncio.WindowsSelectorEventLoopPolicy())
 
-    # Create Rabbit listener
     print("Creating Rabbit Listener")
-    io_loop = tornado.ioloop.IOLoop.instance()
-    rabbit_connection = TornadoAdapter(rabbitmq_url=RABBIT_URI, io_loop=io_loop)
-    rabbit_connection.receive(exchange="test_rpc", routing_key="fib_calc", queue_name="fib_calc_q", handler=handle_message)
+    configuration = dict(
+        receive=dict(
+            exchange="test_rpc",
+            exchange_type="direct",
+            routing_key="fib_calc",
+            queue="fib_calc_q",
+            durable=True,
+            auto_delete=False,
+            prefetch_count=1
+        ),
+        publish=dict(
+            exchange="test_server",
+            exchange_type="direct",
+            routing_key="fib_server",
+            queue="fib_server_q",
+            durable=True,
+            auto_delete=False,
+            prefetch_count=1
+        )
+    )
+    # Using Tornado IO Loop
+    io_loop = tornado.ioloop.IOLoop.current()
+    rabbit_connection = TornadoAdapter(rabbitmq_url=RABBIT_URI, configuration=configuration, io_loop=io_loop)
+    rabbit_connection.receive(handler=handle_message)
```

Comment:
It is not currently possible, since it required a lot of changes and messed up the methods. It is probably possible, though it might need some consideration. We can however make a small change for consuming and open a separate channel for each one (like today)